src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_FOR_DAY = 40;
+  const THREE_DAYS_DISCOUNT = 20;
+  const SEVEN_DAYS_DISCOUNT = 50;
+
+  const THREE_DAYS = 3;
+  const SEVEN_DAYS = 7;
+
+  if (days < THREE_DAYS) {
+    return days * RENT_FOR_DAY;
+  }
+
+  if (days < SEVEN_DAYS) {
+    return days * RENT_FOR_DAY - THREE_DAYS_DISCOUNT;
```

Comment:
Consider using `else if` here to make it clear that this condition is mutually exclusive with the previous one. This can improve readability by explicitly showing that only one of these blocks will execute.